Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.0
 
 FROM busybox as downloader
 
-RUN wget -P /tmp https://github.com/microsoft/ApplicationInsights-Java/releases/download/${APP_INSIGHTS_AGENT_VERSION}/applicationinsights-agent-${APP_INSIGHTS_AGENT_VERSION}.jar
+RUN wget -P /tmp https://github.com/microsoft/ApplicationInsights-Java/releases/download/2.5.0/applicationinsights-agent-2.5.0.jar
```

Comment:
please explain

Reply:
not sure which part to explain so will try to explain as fully as i can.

commit message says "Fix `wget` command in Dockerfile". diff suggests that previous version did not work. ARG setting is not used in this command line run hence need hardcoded yet again.

it is way easier to change 3 places in this file than downloading jar yourself and sticking in repository.

inspected such error on different application - sample app just moments ago

Reply:
Do we need APP_INSIGHTS_AGENT_VERSION any more than ?

Reply:
Can't see in jenkins library. If suppose to be used only by that library then it doesn;t serve any value right now - single point of use below in same file. I can test it out once pipeline is ready in Jenkins. For now - let's keep it as all other projects (including template) has it